package.json

```diff
@@ -33,7 +33,7 @@
     "webpack-dev-middleware": "^1.6.1"
   },
   "peerDependencies": {
-    "@adobe/reactor-turbine": "*"
+    "@adobe/reactor-turbine": ">=18.2.0"
```

Comment:
This assumes that the result of https://github.com/Adobe-Marketing-Cloud/reactor-turbine/pull/5 will be version 18.2.0.